jupyterlab/README.md

```diff
@@ -46,11 +46,11 @@ Then refresh the browser.
 Remove the `pip` package by running:
 
 ```bash
-pip uninstall qsharp_jupyterlab
+pip uninstall qsharp-lang-jupyterlab
```

Comment:
A few files above refer to the package as `qsharp-jupyterlab`.  Which one is it?

Reply:
I was hoping you could confirm all that! I think the issue is that the package name is qsharp-lang-jupyterlab, but the dir we build into is qsharp-jupyterlab. Might want to clean that up later.

Reply:
Oh I can take a closer look then. (I also prefer the package name `qsharp-jupyterlab` but it's OK if that case is closed now.)

Reply:
I think we can decide to change the package name in a follow up. Agree that `qsharp-lang-jupyterlab` is a bit of a mouthful.